### PR TITLE
Adrienne / Reduced bundle size by code splitting svg symbols (SPIKE, DO NOT MERGE)

### PR DESCRIPTION
**Pull request by adrienne-deriv**
Changes:
- Reduce the build bundle size by code splitting imports for symbols and shortcode symbols
Before (tested on `deriv.com/markets/forex`) 4.991MB:

After 2.991MB:

`symbols.ts` and all svgs declared within `src/features/pages/markets/etf/instruments/data.tsx` is included within the build bundle output and takes up a majority of the bundle space, by splitting these svgs into their own individual chunks, we can load them only when needed in their pages:
Before:

After:

Issues to fix:
- SVGs in live price columns in home page is flickering
- ~Caching is somewhat disabled when using this approach~ (This is resolved, the Disable cache Network setting was enabled in my devtools, thus disabling caching)

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/elements/lazy-symbol.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    // name: keyof typeof icons
+    name: string
+    width?: string
+    height?: string
+    alt?: string
+}
+const LazySymbol = ({ alt = 'symbol', name, width = '24', height = '24' }: TLazySymbolProps) => {
+    const [src, setSrc] = React.useState('')
+
+    React.useEffect(() => {
```
Comment (habib-deriv) [inline src/components/elements/lazy-symbol.tsx:14]: Let's import the useEffect from react

File: src/components/elements/lazy-symbol.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import icons from 'components/elements/symbols'
+
+type TLazySymbolProps = {
+    // name: keyof typeof icons
+    name: string
+    width?: string
+    height?: string
+    alt?: string
```
Comment (habib-deriv) [inline src/components/elements/lazy-symbol.tsx:9]: Let's have the `alt` as `required`

File: src/pages/markets/components/helper/_symbol.tsx [hunk 3]
```diff
@@ -1,47 +1,38 @@
...
         <React.Fragment>
             {instruments_type ? (
                 <React.Fragment>
-                    {instruments_type.map(({ src, text }) => (
-                        <SymbolContainer key={text}>
-                            <img src={src} alt={localize('_t_symbol_t_')} />
-                            <Text>
-                                <Localize translate_text={text} />
-                            </Text>
+                    {instruments_type.map((symbol, index) => (
```
Comment (habib-deriv) [inline src/pages/markets/components/helper/_symbol.tsx:15]: Let's use symbol.text as id, can remove the index

File: src/components/elements/lazy-symbol.tsx (hunk 2 above)
Comment (habib-deriv) [reply src/components/elements/lazy-symbol.tsx:9]: Can we make sure, this is only for svg ?

Comment (adrienne-deriv) [reply src/components/elements/lazy-symbol.tsx:9]: Sure, I'll add a check for SVGs

**Pull request closed without merging**